runtime/doc/treesitter.txt

```diff
@@ -1137,8 +1137,13 @@ get_lang({filetype})                      *vim.treesitter.language.get_lang()*
 inspect({lang})                            *vim.treesitter.language.inspect()*
     Inspects the provided language.
 
-    Inspecting provides some useful information on the language like node
-    names, ...
+    Inspecting provides some useful information on the language like node and
+    field names, ABI version, and whether the language came from a WASM
+    module.
+
+    Node names are returned in a table mapping each node name to a `boolean`
+    indicating whether or not the node is named (i.e., not anonymous).
+    Anonymous nodes are surrounded with double quotes (`"`).
```

Comment:
single-quotes since that matches our style guidelines? cc @clason 
```suggestion
    Anonymous nodes are surrounded with quotes (`'`).
```

Reply:
But double quotes is how tree-sitter denotes anonymous nodes -- both in the playground (our `:InspectTree` and in the queries.